reference/platform-app-yaml.md

```diff
@@ -174,7 +174,7 @@ It has a few subkeys which are:
 
 #### Locations
 
-The `locations` key allows you to provide specific parameters for different URL prefixes.
+The `locations` key allows you to provide specific parameters for different URL prefixes. Each entry's key is a domain-root-relative path, and its value is configuration directives for that path.  That is, if your domain is `example.com` then `"/"` means "requests for `example.com/`", while `"/admin"` means "requests for `example.com/admin`".
 
```

Comment:
I think this would be better worded as:
> The `locations` key allows you to customize how your application handles the path component of an incoming request URL. It contains a key-value mapping, where each key is a path prefix that will be compared with the request URL, and the value is another mapping of configuration keys that lets you customize how to handle request that match that path prefix. If more than one location matches the same request URL, the longest matching location is used.
> 
> For example, if your domain is `example.com`, and you have a web configuration with locations set as follows:
> ```
> web:
>     locations:
>         "/":
>             [...]
>         "/a":
>             [...]
>         "/a/b":
>             [...]
> ```
>
> Then request URLs will match as follows:
> - an incoming request with a URL path of `/a` will match the `/a` location
> - `/b` will match the `/` location
> - `/a/b/c` will match the `/a/b` location
> - `/a/c/b` will match the `/a` location
>
> This allows us to set up general URL handling within the `/` location, and
> then use more specific locations to override that behaviour and handle
> certain URL paths differently.